src/test/resources/journeys.conf

```diff
@@ -1,32 +1,38 @@
 # Copyright 2023 HM Revenue & Customs
 #
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # Configure here your journeys. A journey is a sequence of requests at a certain load.
 
 journeys {
-
-  # Example
-  # Give a name to the journey.
   example-journey = {
-
-    # The description will appear in the test report. Use something meaningful
-    description = "Example journey"
-
-    # The load is in journeys per second. Put here the load you are going to have at the peak.
-    # There is no need to put a higher value at this point. Use prerftest.loadPercentage in application.conf instead
-    load = 9.1
-
-    # This points to a csv file with the data you need to inject in the journey. [More here](https://github.com/hmrc/performance-test-runner#step-4-configure-the-user-feeder)
-    feeder = data/example.csv
-
-    # The parts your journey is made of. A part is made one or more requests.
+    description = "Auth Login API Requests"
+    load = 1
     parts = [
-      home-page,
-      post-vat-return-period,
-      get-turnover-page
+      API-Individual,
+      API-Agent,
+      API-Organisation
```

Comment:
Could you please confirm if we need test for API organisation?

Reply:
I was adding these as the mtd-api are testing are doing it, in case we have to do the same